code/datums/controllers/artifact_controls.dm

```diff
@@ -214,8 +214,13 @@ var/datum/artifact_controller/artifact_controls
 
 /datum/artifact_origin/ancient
 	name = "ancient"
-	fault_types = list(/datum/artifact_fault/burn,/datum/artifact_fault/irradiate,/datum/artifact_fault/shutdown,
-	/datum/artifact_fault/murder,/datum/artifact_fault/zap)
+	fault_types = list(
+		/datum/artifact_fault/burn = 10,
+		/datum/artifact_fault/irradiate = 10,
+		/datum/artifact_fault/shutdown = 10,
+		/datum/artifact_fault/zap = 10,
+		/datum/artifact_fault/explode = 10,
+		/datum/artifact_fault/messager/ai_laws)
```

Comment:
`weighted_pick` interprets this as having 0 weight. Maybe that behaviour should be changed and it should interpret it as weight 1?

Reply:
Oh, that is just an oversight! I forgot to put a value for that!